Migrated Expression Resolver test.

cypo721 opened the pull request:
Created ELResolver testcase.

Commit:
Migrated Expression Resolver test.

Created ELResolver testcase.

zebrik commented on basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java:
```diff
@@ -0,0 +1,180 @@
+
+
+
+    /* UTILS */
```
I suggest to extract all UTILS methods to UI abstraction class .. something like `ServerConfigurationPage` to separate the UI and test logic.

zebrik commented on basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java:
```diff
@@ -0,0 +1,180 @@
+    private void goToServerProperties(String server) {
+        FinderNavigation navigation = new FinderNavigation(browser, DomainRuntimeEntryPoint.class)
+                .addAddress(FinderNames.BROWSE_DOMAIN_BY, FinderNames.HOSTS)
+                .addAddress(FinderNames.HOST, "master")
```
Please replace `"master"` with `org.jboss.hal.testsuite.util.ConfigUtils.getDefaultHost()`

Commit:
Utils method extracted to ServerCofigurationPage.

Updated ELresolver test.
Checkstyle fix.
Replaced "master" with ConfigUtils.getDefaultHost().

cypo721 commented:
Updated.

zebrik commented:
Thank you.
